apps/docs/templates/docs/github_detail.html

```
@@ -52,6 +70,55 @@ <h2>{% trans "Respond" %}</h2>
+            <option value="rebase">{% trans "Rebase and merge" %}</option>
+          </select>
+        </div>
+        <div class="mb-3">
+          <label class="form-label" for="commit_title">{% trans "Commit title (optional)" %}</label>
+          <input class="form-control" id="commit_title" name="commit_title" type="text">
+        </div>
+        <div class="mb-3">
+          <label class="form-label" for="commit_message">{% trans "Commit message (optional)" %}</label>
+          <textarea class="form-control" id="commit_message" name="commit_message" rows="4"></textarea>
+        </div>
+        <button class="btn btn-success" type="submit" {% if not merge_allowed %}disabled aria-disabled="true"{% endif %}>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)